### PR TITLE
ABM_05: Updated main screen to have main image

### PR DESCRIPTION
**Pull request by samir-abmcollege**

### CODE REVIEW

**Review by jordanpaulperalta — COMMENTED** [review]
Looks good!

File: Git-Collabration-Example/ViewController.swift [hunk 1]
```diff
@@ -9,9 +9,12 @@ import UIKit
...
 class ViewController: UIViewController {
 
+    @IBOutlet weak var imageView: UIImageView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        imageView.image = UIImage(named: "work")
     }
```
Comment (radiney) [inline Git-Collabration-Example/ViewController.swift:19]: this is a test comment
please delete after

**Review by jordanpaulperalta — APPROVED** [review]

**Review by radiney — APPROVED** [review]

File: Git-Collabration-Example/Assets.xcassets/work.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "work.jpeg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
```
Comment (tonyekas) [inline Git-Collabration-Example/Assets.xcassets/work.imageset/Contents.json:9]: This part looks good

File: Git-Collabration-Example/ViewController.swift [hunk 3]
```diff
@@ -9,9 +9,12 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    @IBOutlet weak var imageView: UIImageView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+
```
Comment (tonyekas) [inline Git-Collabration-Example/ViewController.swift:16]: Asking to have this deleted ... just for test collaboration

File: Git-Collabration-Example/Base.lproj/Main.storyboard [hunk 4]
```diff
@@ -22,18 +22,30 @@
...
                             <constraint firstItem="aWD-dK-fwF" firstAttribute="top" secondItem="6Tk-OE-BBY" secondAttribute="top" constant="70" id="wR5-On-XdB"/>
                         </constraints>
                     </view>
+                    <connections>
+                        <outlet property="imageView" destination="nAc-p1-BI8" id="GqI-He-keG"/>
+                    </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="dkx-z0-nzr" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="139" y="89"/>
```
Comment (tonyekas) [inline Git-Collabration-Example/Base.lproj/Main.storyboard:36]: We try x = 140, y = 90 as a round number

**Review by tonyekas — CHANGES_REQUESTED** [review]
Good training

File: Git-Collabration-Example/ViewController.swift (hunk 1 above)
Comment (samir-abmcollege) [reply Git-Collabration-Example/ViewController.swift:19]: @radiney It's been already deleted

**Comment by tonyekas** [comment]
Is it possible to edit an already submitted comment or delete it completely on the collaboration ..?

File: Git-Collabration-Example/Base.lproj/Main.storyboard (hunk 4 above)
Comment (samir-abmcollege) [reply Git-Collabration-Example/Base.lproj/Main.storyboard:36]: @tonyekas This has been already removed

File: Git-Collabration-Example/ViewController.swift (hunk 3 above)
Comment (samir-abmcollege) [reply Git-Collabration-Example/ViewController.swift:16]: @tonyekas This line has been already removed

File: Git-Collabration-Example/Assets.xcassets/work.imageset/Contents.json (hunk 2 above)
Comment (tonyekas) [reply Git-Collabration-Example/Assets.xcassets/work.imageset/Contents.json:9]: For training purposes, it looks great for us

**Review by tonyekas — APPROVED** [review]
Looks good to me

**Pull request merged**